bin/log-include.sh

```diff
@@ -22,8 +22,8 @@ else
   fi
 fi
 
-if [ "$LOG_DEBUG_ENABLE" != "true" ]; then
-  exec >/dev/null
+if [ "$LOG_VERBOSE_ENABLE" != "true" ]; then
```

Comment:
Should we add a comment here explaining why/what we are doing?  I think having a comment like that here originally would have allowed us to catch this mistake during the initial code reviews.

Reply:
Added comment in latest commit

Reply:
Thank you for the comment.  But, sorry, I meant adding a comment to explain line 26 (the exec 1>/dev/null) since the purpose of that is less obvious.

Reply:
Added the comment